lab-will/server.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const net = require('net');
+const EE = require('events');
+const Client = require('./model/client.js');
+// const PORT = process.env.PORT || 3000;
+
+// Random Port;
+const PORT = Math.floor(Math.random() * ((9999-1000) + 1) + 1000);
```

Comment:
Really?  Every time I restart your server I have to change the port number on telnet? :-)